gradle-plugin/README.md

```diff
@@ -1,16 +1,34 @@
-# Android Product Team's Gradle plugin
+# Mozilla Android gradle-plugin
+The [`org.mozilla.android` gradle plugin, hosted on plugins.gradle.org][plugins], is used to share build code across Mozilla's Android projects like Firefox for Fire TV.
 
-## Publishing
-To publish the plugin to the gradle plugin portal, first [create credentials][]. After making your changes and bumping the version, run the following from the root directory:
-```gradlew
-./gradlew gradle-plugin:publishPlugins
+## Using the plugin
+To use the plugin, you must first apply it and then configure specific tasks
+you want to run. To apply the plugin, use the legacy plugin application method:
+```groovy
+// <project-root>/build.gradle
+buildscript {
+  repositories {
+    maven {
+      url "https://plugins.gradle.org/m2/"
+    }
+  }
+  dependencies {
+    classpath "org.mozilla.android:gradle-plugin:0.1"
+  }
+}
+
+// in the module to use the plugin, e.g. <project-root>/app/build.gradle
+apply plugin: "org.mozilla.android"
 ```
```

Comment:
Nit: I would consider reformatting this into two separate code blocks, to make it more clear that it includes two files.

#### //my-project-root/build.gradle
```groovy
buildscript {
  repositories {
    maven {
      url "https://plugins.gradle.org/m2/"
    }
  }
  dependencies {
    classpath "org.mozilla.android:gradle-plugin:0.1"
  }
}
```

#### //my-project-root/my-module/build.gradle
```groovy
apply plugin: "org.mozilla.android"
```